mm2src/mm2_main/src/rpc/streaming_activations/heartbeat.rs

```diff
@@ -0,0 +1,36 @@
+//! RPC activation and deactivation for the heartbeats.
+use super::{EnableStreamingRequest, EnableStreamingResponse};
+
+use crate::heartbeat_event::{HeartbeatEvent, HeartbeatEventConfig};
+use common::HttpStatusCode;
+use http::StatusCode;
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::{map_to_mm::MapToMmResult, mm_error::MmResult};
+
+#[derive(Deserialize)]
+pub struct EnableHeartbeatRequest {
+    pub config: HeartbeatEventConfig,
+}
```

Comment:
Can't we generalize all these event activation/deactivation mechanism? There will be (which is already happening) a lot of these duplicated-like methods in near future.